### ISSUE TITLE
Input fields' lack of accessibility features.

### ISSUE THREAD

**Issue by GaddamThanmayeeReddy** [prakashmariappan/CallofCreative#8]
<label htmlFor="name-input">Enter Your Name:</label>
<input id="name-input" type="text" placeholder="Enter Your Name" autoComplete="off" value={name} onChange={(event) => setName(event.target.value)} />
 <label htmlFor="bio-input">Enter Your Bio:</label>
<input id="bio-input" type="text" placeholder="Enter Your Bio" autoComplete="off" value={bio} onChange={(event) => setBio(event.target.value)} />

Description: The 'name' and 'bio' input fields lack important accessibility properties like labels and ARIA features. For people who depend on screen readers and other assistive technology, this reduces the form's usability.
